packages/autorest.typescript/test/integration/generated/nameChecker/src/operations/documents.ts

```diff
@@ -92,19 +92,19 @@ export class DocumentsImpl implements Documents {
 
   /**
    * Suggests documents in the index that match the given partial query text.
-   * @param suggesterName The name of the suggester as specified in the suggesters collection that's part
-   *                      of the index definition.
    * @param search$DONotNormalize$Text The search text to use to suggest documents. Must be at least 1
    *                                   character, and no more than 100 characters.
+   * @param suggesterName The name of the suggester as specified in the suggesters collection that's part
+   *                      of the index definition.
    * @param options The options parameters.
    */
   suggestGet(
-    suggesterName: string,
     search$DONotNormalize$Text: string,
+    suggesterName: string,
     options?: DocumentsSuggestGetOptionalParams
   ): Promise<DocumentsSuggestGetResponse> {
     return this.client.sendOperationRequest(
-      { suggesterName, search$DONotNormalize$Text, options },
+      { search$DONotNormalize$Text, suggesterName, options },
```

Comment:
`search$DONotNormalize$Text` ==> Is this name a bug?

Reply:
https://github.com/Azure/autorest.typescript/blob/main/packages/autorest.typescript/test/integration/swaggers/Data.md I think this is changed by those directives.